third_party/txt/src/txt/font_collection.cc

```diff
@@ -30,6 +30,17 @@
 
 namespace txt {
 
+namespace {
+
+// Example characters representing character classes that may require a
```

Comment:
How are these examples?  Are they just characters that happen to be part of the character class, but you don't want/need to include the entire range?  When should we add characters to this list?

Reply:
Yes - these are characters which are part of a class that may not be covered by the platform's default font.

The Minikin font collection is constructed with a list of fonts and doesn't offer a way to search for additional fonts on demand in order to match a character.  This change attempts to build a list of fonts that contain some frequently seen character blocks and add them as backup options if the font collection's primary family doesn't handle a character.

The fallback_characters list should be extended if there is another character class that users expect to see rendered with the default text style but that is not contained in the platform default font.